line-bot-api-client/src/main/java/com/linecorp/bot/client/LineMessagingClientImpl.java

```diff
@@ -369,6 +369,31 @@ public CompletableFuture<GetAggregationUnitNameListResponse> getAggregationUnitN
         return toFuture(retrofitImpl.getAggregationUnitNameList(limit, start));
     }
 
+    @Override
+    public CompletableFuture<Void> validateReply(ReplyMessage replyMessage) {
+        return toFuture(retrofitImpl.validateReply(replyMessage));
+    }
+
+    @Override
+    public CompletableFuture<Void> validatePush(PushMessage pushMessage) {
+        return toFuture(retrofitImpl.validatePush(pushMessage));
+    }
+
+    @Override
+    public CompletableFuture<Void> validateMulticast(Multicast multicast) {
+        return toFuture(retrofitImpl.validateMulticast(multicast));
+    }
+
+    @Override
+    public CompletableFuture<Void> validateNarrowcast(Narrowcast narrowcast) {
+        return toFuture(retrofitImpl.validateNarrowcast(narrowcast));
+    }
+
+    @Override
+    public CompletableFuture<Void> validateBroadcast(Broadcast broadcast) {
+        return toFuture(retrofitImpl.validateBroadcast(broadcast));
+    }
```

Comment:
According to the API reference(https://developers.line.biz/en/reference/messaging-api/#validate-message-objects-of-reply-message), the validate APIs only support message object and don't support userId, reply token, or other objects. I guess this code works since messaging API ignores unrelated json fields but I'm not sure we can use `ReplyMessage` here.

Reply:
If sdk users understand it, I think it's not problem. (in the first place the API itself seems confusing...)

Reply:
ah... hmmmmmmmmm.... good catch.
i need to implement `ValidateReplyMessage` or something like this... maybe.

Reply:
@Yang-33 fixed by [af41e17](https://github.com/line/line-bot-sdk-java/pull/899/commits/af41e17f0abc28648ae98311ad3d75ecf3270879)

Reply:
@Yang-33 could you release this again?

Reply:
Thank you 🙇